### PR TITLE
Petar d

### PR DESCRIPTION
**Pull request by bezanski**

### CODE REVIEW

File: Petar-Dinev.js [hunk 1]
```diff
@@ -2,6 +2,14 @@ function helloEveryone() {
...
-        hobbies: ["running", "hiking", "fitness"]
+        hobbies: ["running", "hiking", "fitness"],
+        town: "Sofia",
     }
-}
\ No newline at end of file
+    let empty = "";
+    for(i=1; i<=10; i++) {
+        empty += i;
+    }
```
Comment (antonos4o) [inline Petar-Dinev.js:11]: I don't understand: What is the point of those lines of code?

**Review by antonos4o — CHANGES_REQUESTED** [review]

File: Petar-Dinev.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        age: 27,
+        hobbies: ["running", "hiking", "fitness"],
+        town: "Sofia",
+    }
+    let empty = "";
+    for(i=1; i<=10; i++) {
+        empty += i;
+    }
+    console.log(empty);
+}
```
Comment (antonos4o) [inline Petar-Dinev.js:13]: Remove this part of the code @bezanski

**Review by antonos4o — CHANGES_REQUESTED** [review]

File: Petar-Dinev.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        name: "Petar",
+        age: 27,
+        hobbies: ["running", "hiking", "fitness"],
+        town: "Sofia",
+    }
+    let empty = "";
+    for(i=1; i<=10; i++) {
+        empty += i;
+    }
+    console.log(empty);
```
Comment (antonos4o) [inline Petar-Dinev.js:12]: Remove this part of the code and make a new pull request

**Review by antonos4o — CHANGES_REQUESTED** [review]

**Review by antonos4o — APPROVED** [review]
approved

**Comment by bezanski** [comment]
Thank you i appreciate it :)

**Pull request merged**